addon/authenticators/jwt.js

```diff
@@ -120,11 +122,11 @@ export default TokenAuthenticator.extend({
 
```

Comment:
@evoactivity Do you have time to implement the fix for this? I am considering forking off, I will be trying to use this feature in an app for this week.